### ISSUE TITLE
t.split is not a function type error

### ISSUE THREAD

**Issue by sam-dm** [kubeapps/kubeapps#1829]
Try to upgrade bitnami wordpress helm chart using kubeapps dashboard. Receiving an error :

Sorry! Something went wrong.

Troubleshooting:

    Check for network issues.
    Check your browser's JavaScript console for additional errors.
    Check the health of Kubeapps components helm status <kubeapps_release_name>.
    Open an issue on GitHub if you think you've encountered a bug.

In the console I am get type error t.s[lit is not a function.

**Comment by absoludity** [comment]
Thanks @sam-dm . Initially I thought this was a duplicate of #1620, but given that it's our own bitnami chart, it's less likely that we've missed some data there (but not impossible). We'll reproduce and let you know.

**Comment by andresmgot** [comment]
Hi @sam-dm, thank you for filling an issue. I cannot reproduce your issue in my environment, can you give us more info so we are able to debug it?

 - Can you post the version of Kubeapps you are using and from which to which version you are upgrading the `bitnami/wordpress` chart?
 - Can you post the screenshot of the console in which you see the error `t.split is not a function`?

**Comment by sam-dm** [comment]
![kubeapps](https://user-images.githubusercontent.com/65372970/86025739-d9967680-ba2e-11ea-8397-36255faea00e.png)

App Version: v1.9.2
Chart Version: 3.5.3

**Comment by sam-dm** [comment]
BTW cannot update the kuebapps too

**Comment by sam-dm** [comment]
I am also receiving this error message:
E0630 09:29:41.296051   15076 portforward.go:385] error copying from local connection to remote stream: read tcp4 127.0.0.1:8080->127.0.0.1:57353: wsarecv: An existing connection was forcibly closed by the remote host.

**Comment by andresmgot** [comment]
okay, so I believe this is an issue because there were some breaking changes in the chart schemas and older versions of Kubeapps are not able to render newer schemas. The issue should go away if you upgrade kubeapps to the latest version using the `helm` cli. Can you try upgrading Kubeapps from a terminal?

**Comment by sam-dm** [comment]
Yes I was able to upgrade the kubeapps through cli : 
helm upgrade kubeapps --namespace kubeapps bitnami/kubeapps --set useHelm3=true

I had to clear cache

and things went fine afterwards

There are other issues now but I guess it is related to the wordpress chart. I will post on their issues and see how things work from this point onwards.

Currently the case with kubeapps is resolved. Thanks for your support
